crypto/src/test/kotlin/net/corda/v5/crypto/SecureHashTests.kt

```diff
@@ -12,12 +12,9 @@ class SecureHashTests {
     @Test
     fun `toHexString Should HEX representation of digest`() {
         val data = "abc".toByteArray()
-        val algorithm = DigestAlgorithmName.SHA2_384.name
+        val algorithm = DigestAlgorithmName.SHA2_384.name;
```

Comment:
This is still kotlin file.

Reply:
To be fair, it doesn't matter if the test files are still written in Kotlin. All that matters are the files in `main`.

Reply:
I should be more specific. I meant to remove `;` as it is still written in kotlin.

Reply:
yes, done